src/assets/i18n/en_US.json

```diff
@@ -2737,6 +2737,7 @@
   "account.user.widget.monthly_spend_limit.label": "Monthly spend limit",
   "account.user.widget.order_spend_limit.label": "Order spend limit",
   "account.user.widget.view_all_users.link": "View all Users",
+  "account.user.edit.heading": "Edit User Profile",
```

Comment:
pls rename the message key: account.user.update_profile.heading
(this is the missing original key name)